build.gradle

```diff
@@ -94,24 +94,3 @@ jacocoTestReport {
 }
 
 test.finalizedBy jacocoTestReport
```

Comment:
rename integration tests with postfix `IT` to `Test`, so in `gradle clean build` it will be started and on travis `gradle build -x test` nope